integration_tests/tests/e2e-tests.ts

```diff
@@ -0,0 +1,227 @@
+import {
+  deleteS3Events,
+  poll,
+  TableNames,
+} from "../../src/handlers/int-test-support/helpers/commonHelpers";
+import { resourcePrefix } from "../../src/handlers/int-test-support/helpers/envHelper";
+import {
+  createInvoiceInS3,
+  createInvoiceWithGivenData,
+} from "../../src/handlers/int-test-support/helpers/mock-data/invoice/helpers";
+import { queryResponseFilterByVendorServiceNameYearMonth } from "../../src/handlers/int-test-support/helpers/queryHelper";
+import {
+  deleteS3Object,
+  listS3Objects,
+} from "../../src/handlers/int-test-support/helpers/s3Helper";
+
+import {
+  generateTransactionEventsViaFilterLambda,
+  retrieveMoreTestDataFromConfig,
+  TestData,
+  TestDataRetrievedFromConfig,
+} from "../../src/handlers/int-test-support/helpers/testDataHelper";
+import { BillingTransactionCurated } from "./billing-and-transaction-view-tests";
+
+const prefix = resourcePrefix();
+const storageBucket = `${prefix}-storage`;
+const standardisedFolderPrefix = "btm_billing_standardised";
+let eventName: string;
+let dataRetrievedFromConfig: TestDataRetrievedFromConfig;
+
+describe("\n Upload invoice to raw invoice bucket and verify billing and transaction_curated view query results matches with expected data \n", () => {
+  beforeAll(async () => {
+    dataRetrievedFromConfig = await retrieveMoreTestDataFromConfig();
```

Comment:
Why retrieve "more" test data? More than what? Can we be specific about the data being retrieved? Maybe `retrieveVendorServiceAndRatesFromConfig`?

Reply:
renamed the function